### PR TITLE
built the mainJS module

### PR DESCRIPTION
**Pull request by SBorghesani**
# Built main.js module

Added  and exported the renderHTML function, and a stateChange event listener. 

## Checklist:

please make sure I have the right syntax and variable names. Let me know if there are any you want changed.

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { WhistlingWillies } from "./WhistlingWhillies.js"
```
Comment (SharifRashed) [inline src/scripts/main.js]: WhistlingWillies ***

**Review by SharifRashed — DISMISSED** [review]

**Comment by SBorghesani** [comment]
fixed the spelling error

**Review by galmodovar — APPROVED** [review]
Looks good to me 👍

**Review by SharifRashed — APPROVED** [review]
Talked about issues needed to be fixed. 2nd time is the charm!

**Pull request merged**